tap_harvest_forecast/__init__.py

```diff
@@ -190,14 +186,20 @@ def sync_endpoint(catalog_entry, schema, mdata, date_fields = None):
                 rec = transformer.transform(row, schema, mdata)
                 append_times_to_dates(rec, date_fields)
 
-                if updated_at >= start:
+                try:
+                    updated_at = utils.strptime_to_utc(rec[REPLICATION_KEY])
+                except KeyError:
+                    updated_at = dateStart
+
+                if start <= updated_at <= dateEnd and rec["id"] not in ids:
```

Comment:
```suggestion
                if dateStart <= updated_at <= dateEnd and rec["id"] not in ids:
```

it seems it should be `dateStart`, insteado of `start`. but for some reason now it returns for me fewer data than before XD. I am investigating that.